deploy/examples/operator-openshift.yaml

```
@@ -570,6 +570,15 @@ data:
   # NOTE: the value here serves as an example and needs to be
   # updated with node labels that define domains of interest
   # CSI_TOPOLOGY_DOMAIN_LABELS: "kubernetes.io/hostname,topology.kubernetes.io/zone,topology.rook.io/rack"
+
+  # Enable read affinity for rbd volumes.
+  CSI_ENABLE_READ_AFFINITY: "false"
+  # CRUSH location labels define which node labels to use
+  # as CRUSH location. This corresponds to the values used in
+  # ceph CRUSH map.
+  # Defaults to all the labels mentioned in
+  # https://rook.io/docs/rook/v1.10/CRDs/Cluster/ceph-cluster-crd/#osd-topology
```

Let's actually use the latest link. I'd rather risk a broken link than an outdated list of topology labels if they change.
```suggestion
  # https://rook.io/docs/rook/latest/CRDs/Cluster/ceph-cluster-crd/#osd-topology
```